xml/depl_nodes.xml

```diff
@@ -1086,7 +1082,7 @@ getent group postgres | cut -d: -f3</screen>
      </para>
      <variablelist>
       <varlistentry>
-       <term><guimenu>Generate (self-signed) certificates</guimenu>
```

Comment:
Better to keep the text within the guimenu tags as the SW string is called like that, otherwise manual and SW do not match. Alternatively, we need to file a bug report to change the SW string.